test/tests/parallel/set3/mta/smtp_delivery.bats

```diff
@@ -102,50 +102,43 @@ function setup_file() {
 }
 
 @test "should successfully authenticate with good password (plain)" {
-  _run_in_container_bash 'nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-plain.txt'
-  assert_success
+  _send_email 'auth/smtp-auth-plain' '0.0.0.0' '25' "${CONTAINER_NAME}" '-w 5'
```

Comment:
3 of the defaults being explicitly specified repeatedly in this test file just to add some timeout args kind of defeats the benefit of the change :sweat_smile: 

Still worthwhile, and in future should be less of an issue once we switch away from `nc`.

Reply:
Haha, you read my mind too - I am currently re-revising it to be way easier. Should be up in 5 mins.

Reply:
Did you identify why a bunch of test cases are failing with the changes in this PR?

Reply:
There seemed to be weird timing issues, but with my current approach (local, still running tests), these do not appear anymore.

Some other errors were me not renaming stuff consistently.

Reply:
Fix: [1397786](https://github.com/docker-mailserver/docker-mailserver/pull/3105/commits/1397786f0467f7ddb19b15e0f0255cb3ad512cf4)

Reply:
The tests failed, I re-ran them and they failed similarly.

Oddly the other PR that got rebased after your first send_email PR merge today is also failing tests: https://github.com/docker-mailserver/docker-mailserver/pull/3006

Reply:
With my latest commit, all is well, isn't it?